<launch>
  <include file="$(find concaveteam)/launch/frames.launch"/>
  <include file="$(find concaveteam)/launch/camera.launch"/>
  <node pkg="concaveteam" type="track_mono" name="track_mono"/>
  <node pkg="concaveteam" type="point2d_to_spherical" name="point2d_to_spherical">
    <param name="~angular_width" value="2"/>
  </node>
</launch>
